### ISSUE TITLE
No Get Started page for easy integration

### ISSUE THREAD

**Issue by karoltarasiuk** [ckeditor/ckeditor5#538]
Hey guys, first and foremost good work there!

I think this is an issue for many of us - we read a lot of awesomeness about CKEditor 5. But as a dev I would like to try it from a different angle, e.g. quickly spin up a page, and start customising :) Yet I can't find a page to help me with it.

Do you think it would be worth adding a subpage here with tutorial to quickly get started https://ckeditor5.github.io/ ?

**Comment by karoltarasiuk** [comment]
I'm very interested in everything happening around RTE space, and at the moment comparing both Draft JS and CKE. As soon with a team we may have to decide to go a step further. And first thing I've noticed is that FB made it much easier to start fiddling around with their editor.

**Comment by Reinmar** [comment]
Hey Karol,

We're in a middle of "writing a basic docs" process. Many things are still missing, so we don't link to the docs officially yet, but they are available on: https://ckeditor5.github.io/docs/nightly/ckeditor5/latest.

The most complete bits are:

* API: https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/api/index.html
* CKEditor 5 builds guides: https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/builds/index.html

   Here you'll find how to install, run and customise (a bit) the editor. 
* Features: https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/features/index.html
* There's also (not yet ported) guide about the dev environment here: https://github.com/ckeditor/ckeditor5/wiki/Development-Environment

There's nothing about the framework yet, unfortunately. That's our next step.

---

[OT]

Regarding Draft.js, I'm sure it's easier to fiddle with. That's no surprise cause it's, in general, a much simpler project. It allowed FB to kick it off much faster and, no matter what we'll do, it will always make Draft.js easier to comprehend. 

CKEditor 5 and Draft.js have a different scope on various levels. CKEditor 5 is not only a framework (Draft.js is mostly oriented on being a framework) but it also contains complete feature implementations, builds, translation services, etc. Also, the [CKEditor 5's engine](https://github.com/ckeditor/ckeditor5-engine) (which is our editing framework) is from the beginning designed to be a much more powerful platform (OT, modularity, composability). But that doesn't come cheap, of course.

tl;dr: To my knowledge, Draft.js and CKEditor 5 are very different projects, focused on meeting different goals. 
tl;dr: I hate to say that, but it's going to take some time before CKEditor 5 has a complete documentation and stable API.

[/OT]

**Comment by Reinmar** [comment]
I confirmed this ticket and added it to it12 to verify later on that finding "quick start" guides is easy.

**Comment by karoltarasiuk** [comment]
Thanks for the answer!

Behind Draft JS we have FB, behind CKE we got a very experienced team as well. There is no other reliable option imo atm, which also is CSP complaint. The only question is which suits us better for the task we want to accomplish, and this is what I'm trying to figure out. Will surely look at the links you've sent.

**Comment by karoltarasiuk** [comment]
Hey @Reinmar, about Plugin Development Guide mentioned here: https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/builds/guides/development/plugins.html#Creating-plugins: are there any drafts or some working version of it?

**Comment by Reinmar** [comment]
No, unfortunately, not yet. I'll be working on framework guides next week. This will be one of the first ones. I'll let you know once something's out.

**Comment by Reinmar** [comment]
Hey, small update. This week, I wrote 4 new guides:

* [Framework overview](https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/framework/guides/overview.html)
* [Quick start](https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/framework/guides/quick-start.html) guide with some most basic example of a plugin.
* [Introduction to framework architecture](https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/framework/guides/architecture/intro.html) (not yet complete)
* [Advanced setup](https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/builds/guides/integration/advanced-setup.html) – few more words about building and integrating CKEditor.

They haven't been read even once yet so expect a lot of mistakes, but the meat is there already.

I hope this helps at least a bit :)

**Comment by Reinmar** [comment]
OK, the quick start guide and intro to the framework have been extended. We also documented more features and packages. There's still a lot to do and there will always be. But we can close this issue for now. Please report new tickets if you miss some specific information.
